src/IO/S3/PocoHTTPClient.cpp

```diff
@@ -497,12 +497,12 @@ void PocoHTTPClient::makeRequestInternalImpl(
                     LOG_TEST(log, "Written {} bytes to request body", size);
             }
 
+            setTimeouts(*session, getTimeouts(method, first_attempt, /*first_byte*/ false));
```

Comment:
that replacement I have made because GCP is often goes out of tight first byte timeout when answering. That does not ruin concept just because now it means just first any byte. Before there was some distinguishing between send/receive.